packages/contracts/contracts/analytics-utilities/libs/QueryTrancheHelpers.sol

```diff
@@ -70,9 +70,10 @@ library QueryTrancheHelpers {
 
         for (uint8 i = 0; i < assets.length;) {
 
-            QueryAssetHelpers.AssetData memory assetData =
-                QueryAssetHelpers.getAssetData(assets[i], tranche, addressesProvider, address(0));//TODO: add chainlink address if this function is needed
+            QueryAssetHelpers.AssetSummary memory assetData =
+                QueryAssetHelpers.getAssetData(assets[i], tranche, addressesProvider);
 
+            // TODO: convert amounts to usd
```

Comment:
is this still relevant?

Reply:
Yes. it doesn't make sense to be adding raw values of different tokens up like that. Currently TrancheHelpers are not used so this is not fixed. however if we want to use the tranche eth call functions, we need to fix this.